cmake/external/leveldb.cmake

```diff
@@ -26,6 +26,7 @@ ExternalProject_Add(
   DOWNLOAD_DIR ${FIREBASE_DOWNLOAD_DIR}
   DOWNLOAD_NAME leveldb-${version}.tar.gz
   URL https://github.com/google/leveldb/archive/${version}.tar.gz
+  URL_HASH SHA256=9a37f8a6174f09bd622bc723b55881dc541cd50747cbd08831c2a82d620f6d76
```

Comment:
is this the SHA256 of `https://github.com/google/leveldb/archive/1.23.tar.gz` ?

I got `a1c64afd13a4a4e3cc5a4d8e04d7b2ec4c95208bd2b51b29e583598da5675b67` 😄

Reply:
Huh, can you try again? I got `9a37f8a6174f09bd622bc723b55881dc541cd50747cbd08831c2a82d620f6d76`

Here is what I saw (with some noise removed):

```
$ wget https://github.com/google/leveldb/archive/1.23.tar.gz
HTTP request sent, awaiting response... 200 OK
Length: 242925 (237K) [application/x-gzip]
Saving to: ‘1.23.tar.gz’

$ sha256sum 1.23.tar.gz
9a37f8a6174f09bd622bc723b55881dc541cd50747cbd08831c2a82d620f6d76  1.23.tar.gz
```

Reply:
oops 🤦‍♂️  sorry. it's correct.

Reply:
no problem. that is one thorough review! thanks!